lib/msal-browser/src/controllers/IController.ts

```diff
@@ -102,9 +103,12 @@ export interface IController {
 
     setNavigationClient(navigationClient: INavigationClient): void;
 
+    getRequestCorrelationId(request?: Partial<BaseAuthRequest>): string;
```

Comment:
We can rename it to something like `generateCorrelationId` if preferred. Suggestions welcome. This change is to enable PWB or any app to call this API to fetch the request correlationId or generate a GUID.

Reply:
We can also just expose GUID generation API and restrict the public API to the same.

Reply:
Let me do a different PR for correlation-id related fixes, to track this for RT use case.